lib/rules/lint-no-index-component-invocation.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const Rule = require('./base');
+
+module.exports = class LintNoIndexComponentInvocation extends Rule {
+  visitor() {
+    const sourceForNode = this.sourceForNode.bind(this);
+    const log = this.log.bind(this);
+    function lintIndexUsage(node) {
```

Comment:
if you make this a method on the `LintNoIndexComponentInvocation` class you don't have to explicitly bind the methods above 😉

Reply:
using function make easier to test logic in astexplorer